### ISSUE TITLE
Patterns > synced patterns > cannot unsync

### ISSUE THREAD

**Issue by Marc-pi** [WordPress/gutenberg#66774]
## What problem does this address?

I can create my own custom patterns, but once those are created with the synced flag to ON, i cannot undo once submitted
the only way it to create a new instance of my pattern or detach it each time it is used 

Just as a reminder, by default a new pattern is created with the flag set to ON

Tested with WP6.7RC3 + TT4 + GB19.5.1

![Image](https://github.com/user-attachments/assets/7116aaa2-9304-466e-9a69-6e55ce5b5716)

## What is your proposed solution?

* In the patterns list page, add a way to unsync in the 3 dots menu, if the pattern is not used somewhere or is just created

mysite.com/wp-admin/site-editor.php?categoryId=my-patterns&postType=wp_block

![Image](https://github.com/user-attachments/assets/84654520-851b-4de3-a00e-7fb5cf7e4dc8)

* once edited, permit to change the status
 ![Image](https://github.com/user-attachments/assets/5d07bc6e-5d6d-4522-88bc-a363ff1b5c32)

**Comment by talldan** [comment]
I think this has been discussed before in another issue or maybe it was when the sync status was first introduced, but I'll recap it here. 

The challenge with being able to change the sync status is knowing what to do with existing instances of the patterns that have been inserted in various posts, pages, templates etc.

If the pattern's source were changed from 'Unsynced' to 'Synced' - any existing pattern instances would all be 'unsynced'. It might be ok to leave them as 'unsynced', it'd be similar to if the blocks were detached from the synced version of a pattern. Some users though might expect that all of these instanced are converted to 'Synced' pattern instances.

I think it's worse when the pattern's source is changed from 'Synced' to 'Unsynced'. The user might already have lots of synced instances inserted across their site. If the user changes the blocks in the pattern source, those synced instances will all update even though the source pattern has now been marked as 'unsynced'.

There might be ways to solve it. The two that come to mind are bulk conversions of blocks across a site, or alternatively linking synced patterns to only the 'synced' revisions of a pattern source.

> In the patterns list page, add a way to unsync in the 3 dots menu, if the pattern is not used somewhere or is just created

I saw you mentioned this, so perhaps you'd already considered the above problem. At the moment there's no way to tell if a pattern is used, though it is another tracked feature (https://github.com/WordPress/gutenberg/issues/60205), so once that's solved it might be possible!

Right now the quickest alternative seems to be using duplicate and then deleting the old pattern.

**Comment by Marc-pi** [comment]
> I saw you mentioned this, so perhaps you'd already considered the above problem.

exactly, in "if the pattern is not used somewhere or is just created"
for me changing the sync status is only doable if the pattern is unused yet in any content

> At the moment there's no way to tell if a pattern is used, though it is another tracked feature (https://github.com/WordPress/gutenberg/issues/60205), so once that's solved it might be possible!

good catch, did not see this issue, it is a prerequisite, 

> Right now the quickest alternative seems to be using duplicate and then deleting the old pattern.

yep
thx
